### ISSUE TITLE
Community.

### ISSUE THREAD

**Issue by arthurwolf** [BrainJS/brain.js#846]
Is there some sort of community for users to ask questions?

I have a project, and would like to discuss it with other users. Is there a mailing list, telegram channel, social media, anything like this?

I couldn't find one in the Readme if there is one.

If there isn't one, is an issue the right place to do this instead?

Thanks!

**Comment by Vectorrent** [comment]
I agree, an official community would be great. I've spent so much time beating my head against the wall, because I don't want to bother the maintainers with my trivial issues.

It seems like everyone's using Discord, these days. IPFS, SurrealDB, Logseq, Midjourney... just to name a few I frequent. 

I've taken the liberty of creating an unofficial "official Brain.js Discord server" here: https://discord.gg/N3dsVuWfJr

If it grows, and @robertleeplummerjr or one of the other maintainers want to be involved, I'd be more than happy to transfer ownership to the appropriate parties.

Come join us, I'll answer questions, if I can!

**Comment by splurgebudget** [comment]
Note: they have enabled the `Discussions` area here on GitHub: https://github.com/BrainJS/brain.js/discussions .  Should be able to ask/answer questions there.
